nextflow_schema.json

```diff
@@ -44,6 +28,11 @@
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "analysis": {
+                    "type": "string",
+                    "fa_icon": "fas fa-cog",
+                    "enum": ["screening", "targeted"]
+                },
                 "email": {
                     "type": "string",
                     "description": "Email address for completion summary.",
```

Comment:
I guess this comes from the template. Just had the idea that sth like: 
`Provide your email for receiving a notification about the completion of your pipeline run.` 
would be easier to understand.